### DISCUSSION TITLE
官方的v4接口变了，有人知道新的key吗

### DISCUSSION THREAD

**Discussion by kxy000** [xiaoyaocz/flutter_dmzj#147] [General]
_新的接口 https://nnv4api.idmzj.com/v1/comic/detail/9603  变成这个形式的了，解密的key貌似也变了。 

大家有知道key的吗，
原来json的接口漫画数据如果有番外之类的貌似没数据_

更新：

v4接口这么用可以的 https://nnv4api.idmzj.com/comic/detail/9603?uid=2665531

**Comment by Schezuk** [comment]
>有番外之类的貌似没数据

这是因为fallback到v1api了吧。

**Reply by kxy000** [reply]
> > 有番外之类的貌似没数据
> 
> 这是因为fallback到v1api了吧。

对 应用目前使用的是v1api  ...

**Reply by Schezuk** [reply]
这api挂了之后漏出来了奇怪的东西

```
object(CI_DB_mysql_result)#18 (8) { ["conn_id"]=> bool(false) ["result_id"]=> NULL ["result_array"]=> array(0) { } ["result_object"]=> array(0) { } ["custom_result_object"]=> array(0) { } ["current_row"]=> int(0) ["num_rows"]=> NULL ["row_data"]=> NULL } {"result":1,"msg":"OK","data":{"info":{"cover":"http:\/\/images.dmzj.com\/"},"list":[],"alone":[],"similar":[]}}
```

**Reply by Schezuk** [reply]
> > > 有番外之类的貌似没数据
> > 
> > 
> > 这是因为fallback到v1api了吧。
> 
> 对 应用目前使用的是v1api ...

有没有可能打他一下看看

**Comment by Schezuk** [comment]
key似乎确实不一样了。

**Comment by Schezuk** [comment]
说起来这个应用是不是配了严格的证书信用链？https抓包抓不到……

**Comment by kxy000** [comment]
我弄错了，v4接口可以这么用  https://nnv4api.idmzj.com/comic/detail/9603?uid=2665531
